ex4/VendingDP.Lib/ItemDetectors.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace VendingDP.Lib
+{
+    public static class ItemDetectors
+    {
+        public static CombinatedProduct DetectProduct(string productName) 
```

Comment:
This is a "static-class", all the functions here are static and I get it why, but there are people that would rather not write any static functions at all because they are hard to test, I'm one of those. Moreover you should not care how much objects are created, the garbage collector takes care of it.

Reply:
Resolved